test/util/IdTableHelpers.h

```diff
@@ -81,6 +92,18 @@ IdTable makeIdTableFromVector(const VectorTable& content,
   return result;
 }
 
+// Create IdTables from a vector of VectorTables, where each VectorTable
+// represents a block
+inline std::vector<IdTable> createLazyIdTables(
+    const std::vector<VectorTable>& blocks) {
+  std::vector<IdTable> tables;
+  tables.reserve(blocks.size());
+  for (const auto& block : blocks) {
+    tables.push_back(makeIdTableFromVector(block, ad_utility::testing::IntId));
+  }
+  return tables;
```

Comment:
```suggestion
return ::ranges::to<std::vector>(blocks | ql::views::transform(lambdaThatTurnsTheBlockToAnIdTable))
```

(again needs the `backports/algorithm.h` header.